### DISCUSSION TITLE
Gigabit Switch?

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#710] [Q&A] [answered]
I see something that looks like (and is labelled) a 1 Gb Ethernet switch:

![image](https://user-images.githubusercontent.com/6830886/230374185-eed6d54a-50ad-477f-9afc-0d6510965fb5.png)

What's on enet besides the pi?

**Comment by rstrouse** [comment, marked as answer]
This also connects to the my IntelliCenter as well as a DMX lighting controller.
